### ISSUE TITLE
loss function for different labels 

### ISSUE THREAD

**Issue by betterze** [keisen/tf-keras-vis#3]
Dear Keisen, 

Thank you for sharing with us this nice project. I really like it. 

In attentions.ipynb, cell 4,

~~~~
# Define loss function. 20 is the imagenet index corresponding to ouzel.
loss = lambda output: K.mean(output[:, 20])
~~~~

This loss function is only for images with the same class (20). If I want to use images with different classes (for example, the first image is class 20, the second image is class 21), how should I modify the code?

Thank you for your help.

Best Wishes,

Alex

**Comment by keisen** [comment]
Hi, @betterze .
I’m happy that you like it.

> for example, the first image is class 20, the second image is class 21

If you know the number of images (that’s TWO in above example), you can implement as follows:

```
loss = lambda output: (output[0, 20] + output[1, 21]) / 2
```

Thanks!

**Comment by betterze** [comment]
thx a lot, I understand it now.
